### ISSUE TITLE
Trigger a new harvesting for the Swedish catalog

### ISSUE THREAD

**Issue by fredper-lm** [INSPIRE-MIF/helpdesk-geoportal#207]
Hi,
Clould you please trigger a harvesting for the Swedish node and publish the result?
Best Regards
Fredrik Persäter

**Comment by jescriu** [comment]
Dear @fredper-lm,
At the end of July (concretely on 22 July) we successfully harvested SE endpoint and published the results in the Geoportal frontend, with the follwing results:
![image](https://github.com/user-attachments/assets/67d32f76-a953-46c9-bd19-2f763a54cf7c)
I just re-harvested today (23 August) your endpoint and published the results in the frontend:
![image](https://github.com/user-attachments/assets/1e914ec6-18e9-4952-ae06-82dd518be899)
Almost the same results.

**Comment by fredper-lm** [comment]
Thanks Jordi!

Do you frequently harvest all countries nodes without us having to request a harvest? If this is the case, I would like a notice when a harvest has been made. This is important in our communication with our agencies. It would also be good to have a report sent out, as above, in connection with the harvest. However, unless there is an automatic process for such a dispatch to all countries, it may be unreasonable.

/Fredrik

**Comment by jescriu** [comment]
Dear @fredper-lm,

In July, I exceptionally did a general harvest of almost all endpoints, since we solved: 
1. The issue regarding the publication of results in the frontend.
2. An issue regarding the GeoNetwork indexing files, which was affecting the indicators for diferrent INSPIRE Themes (See https://github.com/INSPIRE-MIF/helpdesk-geoportal/issues/170#issuecomment-2213269594).
Additionally, it had passed quite a long time since the last harvestings (in most cases being the one we did for the INSPIRE Monitoring 2023).

I still have not been able to warn all harvested Member States / EFTA countries in July. Apologies for this, but I am alone covering all this work (managing improvements, executing and reporting them) for all endpoints, and I also need to cover other businesses.

Hope the system is progressively improved and we can report in a more consistent way in the near future.

Thank you for your understanding.
